packages/azure-kusto-data/package.json

```diff
@@ -18,7 +18,7 @@
         "http": false,
         "crypto": false,
         "stream": false,
-        "./dist-esm/src/connectionBuilder.js": "./dist-esm/src/connectionBuilder.browser.js"
+        "./src/connectionBuilder": "./src/connectionBuilder.browser"
```

Comment:
Im really don't think we should do this
we talked about it - we are going to move to microsoft sdk repo
If this is in order for the version.ts to use import pkg from "../package.json";
i think i have another way  - which is simply to move version.ts one folder up ... didnt try it but i think it would work

Reply:
Even if we move - I don't see the problem here.

The change is to work strictly on typescript files in both webpack and node, as opposed to typescript in node and generated js in webpack.

The change also generates .js files that you can include and use without plugging in to our build system (but you can still do it the old way if you want to).

As for your question above - you're right, we don't need the webpack js files - we again operate entirely on ts files.

The big advantage is not having to worry if the code was compiled, and dealing with generated code.

Moving it a folder up wouldn't matter if you don't include some way to have the data